src/user-event/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    // the wait in the loop is intentional and we don't want to parallelize
```

Comment:
I copy/pasted that comment from one situation, but there are several. I couldn't think of a single situation in this directory where we'd want the parallization so it made sense to me to just disable it for this directory.

Reply:
But this will also apply to any future file/addition. It's intentional for the code right now. But nobody will remember this rule exists if they add new code or it isn't intentional anymore.

Reply:
I honestly can't imagine a situation where we'd want parallelization. The async nature of this directory is to simulate the user's behavior (user's can't do two things at once). I think it's more sensible to disable this rule in this directory rather than add comments everywhere. They're just noise and make people feel like they may be doing something worng.